zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/QuorumSSLTest.java

```diff
@@ -224,12 +228,24 @@ public OCSPHandler(X509Certificate revokedCert) {
         public void handle(com.sun.net.httpserver.HttpExchange httpExchange) throws IOException {
             byte[] responseBytes;
             try {
+                String uri = httpExchange.getRequestURI().toString();
+                LOG.info("OCSP request: {} {} {}", httpExchange.getRequestMethod(), uri, httpExchange.getRequestHeaders().entrySet());
+                httpExchange.getRequestHeaders().entrySet().forEach((e) -> {
+                    LOG.info("OCSP request header: {} {}", e.getKey(), e.getValue());
+                });
```

Comment:
This logs each header twice: once at the end of the `OCSP request: ` line, in "compressed form," and once per line in the rest.  We could perhaps drop the third format argument from the first line?